src/stan-users-guide/for-bugs-users.Rmd

```diff
@@ -292,24 +293,24 @@ They are useful for defining iterative functions with complex
 termination conditions.  As an illustration of their syntax,
 the for-loop
 
-```
+```stan
 model {
-    ....
+    /// ...
```

Comment:
I don't think we ever want three slashes.  Also, I like these not as comments.  Is the reason you're commenting them for formatting in pretty printing?  If so, this probably needs to be done everywhere.